### ISSUE TITLE
Properly implement xs data array input

### ISSUE THREAD

**Issue by howland** [howland/pyne#12]
Need to figure out what form it should look like if there are multiple groups, then fix below!
https://github.com/howland/pyne/blob/transport-josh-dev/src/ahot/spatial_solvers_3d/source/readxs.f90#L66

**Comment by rachelslaybaugh** [comment]
tl;dr: I am not sure that this is what we want to pursue at all. We'd like to use the cross section information already available in PyNE rather than write another parser. 

Part of the motivation of wrapping these solvers to begin with is to be able to leverage what PyNE has - and access to nuclear data is a huge part of that. That was probably not clear, or we talked about it quite a long time ago.

The explanation anyway:
The way I would format the cross sections is to have each material's entries separated by a keyword that goes with a material name, e.g.

material hydrogen
data
data
material graphite
data 
data

or something like that.

For multigroup, you could have each line be its own group, and within a line you would have to select how to structure the data. If you wanted to allow only basic things, I would suggest something like
total absorption fission scattering (where scattering is a complicated thing we can talk about more broadly)

Groups are ordered from high to low energy.

**Comment by howland** [comment]
"Part of the motivation of wrapping these solvers to begin with is to be able to leverage what PyNE has - and access to nuclear data is a huge part of that. That was probably not clear, or we talked about it quite a long time ago."

Oh - ok.  I was under the impression that we would be taking in an array of data that the user put together using the pyne xs data format, that makes much more sense now.  So we will take in the names and find the data using pyne?

I'll back out the changes I made to the dictionary so that the array xs is no longer supported.
